packages/components/button/__tests__/button.test.tsx

```diff
@@ -300,4 +300,23 @@ describe('Button Group', () => {
     const btn = wrapper.findComponent(Button)
     expect(btn.classes()).toContain('el-button--small')
   })
+
+  it('use custom tag disabled click not triggered', async () => {
+    const isLoaing = ref(true)
+    const wrapper = mount(() => (
+      <div>
+        <Button
+          tag="div"
+          loading={isLoaing.value}
+          disabled={!isLoaing.value}
+        ></Button>
+      </div>
+    ))
+    const btn = wrapper.findComponent(Button)
+    await btn.trigger('click')
+    expect(wrapper.emitted('click')).toBeUndefined()
+    isLoaing.value = true
```

Comment:
> L05 `const isLoaing = ref(true)`

IMO, this test might be more complete if the click event is triggered once

Reply:
`    const isLoaing = ref(false);
    const isDisabled = ref(false)`
    two ref, end all set false triggered , How about this ?

Reply:
cool

Reply:
ok,  I add a new commit